README.md

```diff
@@ -9,6 +9,9 @@
 
 [![Build Status](https://travis-ci.org/flutter/sentry.svg?branch=master)](https://travis-ci.org/flutter/sentry)
 [![pub package](https://img.shields.io/pub/v/sentry.svg)](https://pub.dev/packages/sentry)
+[![likes](https://badges.bar/sentry/likes)](https://pub.dev/packages/sentry/score)
+[![popularity](https://badges.bar/sentry/popularity)](https://pub.dev/packages/sentry/score)
+[![pub points](https://badges.bar/sentry/pub%20points)](https://pub.dev/packages/sentry/score) 
```

Comment:
How do these work? Do these update automatically?

Reply:
Code is here: https://github.com/bruno-garcia/badges.bar
It sets `request.response.headers.add('Cache-Control', 'max-age=3600');` but so far on each requests it fetches the latest value.
Also, it's behind cloudflare so for that duration cloudflare should return the value and avoid hitting the service.

I'll add some in-memory cache for the `score` value later to avoid fetching on each request.